lib/commands/crafter/add_office.rb

```diff
@@ -21,41 +21,46 @@ def prepare(data)
       @apprentice = Apprentice.profile(data[:user_id])
     end
 
-    def parse_office_location(data)
-      data[:user_message].gsub("office:", "").strip.downcase
+    def run
+      return ""                 unless user
```

Comment:
What is the variable user at this stage?

Reply:
it's a method call to https://github.com/fabientownsend/lunchbot/blob/bfda70732d410480fb78cf19c2d32939fc2d217d/lib/commands/crafter/add_office.rb#L43

```
    def user
      @crafter || @apprentice
end
```